src/tests/unit/tests/DetailsView/components/export-dialog.test.tsx

```diff
@@ -52,35 +49,30 @@ describe('ExportDialog', () => {
         const isOpenOptions = [true, false];
 
         it.each(isOpenOptions)('with open %p', isOpen => {
-            provideBlobMock
-                .setup(p => p(It.isAny(), It.isAnyString()))
-                .returns(() => blobStub)
-                .verifiable(Times.once());
-            windowUtilsMock
-                .setup(w => w.createObjectURL(blobStub))
+            props.isOpen = isOpen;
+            fileProviderMock
+                .setup(f => f.provideURL(It.isAny(), It.isAnyString()))
```

Comment:
nit: you can probably  use `blobStub` here to do a strick typecheck than `It.isAny()`